shared-components/tl-tournament-card/tl-tournament-card-css.js

```diff
@@ -0,0 +1,79 @@
+import { css } from 'lit';
+
+export const tournamentCardStyles = css`
+    .details-button {
+        justify-content: center;
+        align-items: center;
+        width: 10em;
+        min-height: 4em;
+    }
+    .image-date-wrapper {
+        position: relative;
+    }
+    #date-time-card {
+        position: absolute;
+        top: 0;
+        right: 0;
+        margin: 0 3.6em;
+        z-index: 1;
+    }
+    .tournament-img {
+        position: absolute;
```

Comment:
Why is it with `position: absolute;`? I don't see the value...

I think the image style should be left alone, with the `margin-top` value being transferred to the parent (via the style `.image-wrapper`) as a `padding-top` setting...

Reply:
Unfortunately, something I missed while refactoring. Definitely should be like you mentioned.